### PR TITLE
fixed the styling of get started button

### PR DESCRIPTION
**Pull request by WiseyXD**
241007_22h11m25s_screenshot

fixes https://github.com/DiceDB/playground-web/issues/60#issue-2570939771

### CODE REVIEW

**Comment by ayushsatyam146** [comment]
Hi @WiseyXD please resolve conflicts

**Comment by WiseyXD** [comment]
Hello @ayushsatyam146 conflicts resolved

File: next-env.d.ts
Comment (lucifercr07) [inline next-env.d.ts:1]: Is this file required to check in?

**Review by lucifercr07 — APPROVED** [review]
